Gems/RosRobotSample/gem.json

```diff
@@ -0,0 +1,27 @@
+{
+    "gem_name": "RosRobotSample",
+    "display_name": "RosRobotSample",
+    "license": "Apache-2.0 or MIT",
+    "license_url": "https://opensource.org/licenses/Apache-2.0",
+    "origin": "Ros2WarehouseDemo",
+    "origin_url": "https://github.com/RobotecAI/Ros2WarehouseDemo/tree/main/Gems/RosRobotSample",
+    "type": "Asset",
+    "summary": "This project contains a sample robot asset",
+    "canonical_tags": [
+        "Gem"
+    ],
+    "user_tags": [
+        "RosRobotSample"
+    ],
+    "platforms": [
+        ""
+    ],
+    "icon_path": "preview.png",
+    "requirements": "This requires XO3DE ROS2 Gem (https://github.com/RobotecAI/o3de-ros2-gem)",
+    "documentation_url": "",
+    "dependencies": [
+        "ROS2"
+    ],
+    "repo_uri": "",
+    "restricted": "RosRobotSample"
```

Comment:
Does this gem have a restricted platform? I don't think it does, if not this restricted line should be removed.

Reply:
Done